src/helpers/transpileContactToCozy.js

```diff
@@ -54,4 +54,4 @@ const transpileToCozy = (contact, contactsAccountsId) => {
   }
 }
 
-module.exports = transpileToCozy
+module.exports = transpileContactToCozy
```

Comment:
I think this kind of code should use models.

```jsx
import CozyContact from 'cozy-doctypes/contacts'

class Contact extends CozyContact {
   fromToutatice (toutaticeContact) {
      ...
   }
}
```

Reply:
@y-lohse Do you have an idea to implement this reco?

Reply:
At this point, not really no. I'm not clear about where we can use the model here, or what amount of work it requires.
While I agree we need to start using models, I also think this should be identified and discussed before hand. Trying to retro-fit it into a PR that's essentially finished is not going to be easy, so unless you consider it blocking, I would suggest we leave this to a separate task/PR.